docs/data/sdks/next-gen-sdks.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Next Generation SDKs
+description: Introduction to Next Generation SDK, common methods and benefits
+---
+
+This article helps you remove confusion around various Amplitude Analytics SDKs and introduces the Next Generation SDKs.
+
+The **Next Generation (next-gen) SDKs** are a group of Amplitude Analytics SDKs that share the same architecture and interfaces. 
+
+Amplitude also has "**Maintenance SDKs**". These are the time tested, stable SDKs of old. Once a next-gen SDK is available for a given platform, older SDKs are moved to maintenance status and under the **Maintenance SDKs** section in the site navigation. For example, Browser, Android, React Native, and Node.js. Maintenance SDKs only receives bug fixes until deprecation. It's strongly recommended to adopt next-gen SDKs to take advantage of their benefits. To migrate from a maintenance SDK, refer to the migration guide under each next-gen SDK documentation. 
+
+!!!note 
+    There are still some SDKs that aren't nex-gen while also not in maintenance, for example Flutter, Java, and iOS. 
+
+Refer to the [SDK status table](./#sdk-status-table) to check whether a SDK is next-gen or in maintenance. 
+
+## Architecture
+
+Next-gen SDKs share the same architecture and interfaces across platform. Refer to specific next-gen SDK documentation for more details. 
+
+![Next-gen Architecture](/../assets/images/data-next-gen-sdk-architecture.drawio.svg)
+
+### Client
+
+Amplitude Client stores configurations and is also the entry point for developers to track events. 
+
+Before instrumenting events, you must initialize a SDK client instance using the API key for your Amplitude project. 
+
+```python 
+# Next-gen Python SDK example: initialize a Client
+
+from amplitude import Amplitude
+
+amplitude = Amplitude('API_KEY')
+```
+
+### Configuration
+
+Configuration includes the options to configure Client behavior. 
```

Comment:
Should we talk about the difference between the maintenance SDK and the next gen SDK? 
Most legacy SDKs are using setters, but new SDKs can set the configurations during the init step which is much more convenience.

Reply:
I was thinking to keep this doc mostly focused on NextGen SDK so it can act as the go to reference for how the new SDKs work. I want limit comparison to the legacy SDKs to a single section as not to confuse new users.

Reply:
Discussed this in team sync and we all agreed that we should talk minimum about maintenance SDK in this page, because this page should focus on next-gen SDKs and we do no want our customers to learn something they don't need if they are new to Amplitude